backend/codesirius/problems/serializers/problem.py

```
@@ -0,0 +1,113 @@
+        required=True,
+        error_messages={
+            "required": "The title field is required.",
+            "blank": "The title field must not be blank.",
+            "null": "The title field must not be null.",
+            "max_length": "The title field must not exceed 200 characters.",
+        },
+        validators=[
+            UniqueValidator(
+                queryset=Problem.objects.all(),
+                message="Problem with this title already exists.",
+            )
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)